src/Cake.Core/CakeEnvironment.cs

```diff
@@ -81,6 +82,12 @@ public string GetEnvironmentVariable(string variable)
                     dictionary => dictionary);
         }
 
+        /// <inheritdoc/>
+        public IDictionary<string, string> GetArguments()
```

Comment:
- [ ] Move implementation to [CakeArguments](https://github.com/cake-build/cake/blob/v1.1.0/src/Cake.Core/CakeArguments.cs#L10)
- [ ] Return type should be `IDictionary<string, ICollection<string>>` to allow for multiple values for the same argument

Reply:
Done